overlord/snapstate/autorefresh.go

```diff
@@ -227,12 +243,13 @@ func (m *autoRefresh) launchAutoRefresh() error {
 }
 
 func resetRefreshScheduleToDefault(st *state.State) (ts []*timeutil.Schedule, scheduleStr string) {
-	refreshSchedule, err := timeutil.ParseLegacySchedule(defaultRefreshSchedule)
+	refreshSchedule, err := timeutil.ParseSchedule(defaultRefreshSchedule)
 	if err != nil {
 		panic(fmt.Sprintf("defaultRefreshSchedule cannot be parsed: %s", err))
 	}
 	tr := config.NewTransaction(st)
-	tr.Set("core", "refresh.schedule", defaultRefreshSchedule)
+	tr.Set("core", "refresh.timer", defaultRefreshSchedule)
+	tr.Set("core", "refresh.schedule", defaultLegacyRefreshSchedule)
```

Comment:
Yeah, we definitely don't want that.

Reply:
IIUC neither option should be `tr.Set()` in this case?